HACKING.md

```diff
@@ -396,6 +412,148 @@ multipass delete juju
 multipass purge
 ```
 
+### JIMM controller
+
+First, create a new Multipass container. You may need to adjust the resources
+depending on your host machine, but you will need to allocate at least 20GB of
+disk space.
+
+```shell
+multipass launch --cpus 2 --disk 20G --memory 8G --name jimm
+```
+
+Copy your ssh key into the container. You can do this manually or use this one-liner:
+
+```shell
+cat ~/.ssh/id_rsa.pub | multipass exec jimm -- tee -a .ssh/authorized_keys
```

Comment:
I have `ed25519` keys instead, so maybe we could also add a comment about changing `id_rsa.pub` to `id_ed25519.pub` if needed?

Reply:
Done.